Update README for bitcoindesign/guide repo (#120)
BitcoinDesign/Guide: README.md

# Bitcoin Design Guide

> This is a project overview of a design guide for bitcoin with early thoughts on what it could end up being.

Reviewer: I kinda like this story adds a bit of personality. :smile:

Reviewer 2: travel guide story is in the design guide itself now

> This is the repository for the [Bitcoin Design Guide](https://bitcoin.design/guide/), and the [bitcoin.design](http://bitcoin.design) community website.

Reviewer 3: ```suggestion
This is the repository for the [Bitcoin Design Guide](https://bitcoin.design/guide/), and the [bitcoin.design](https://bitcoin.design) community website.
```

Discussion and collaboration is taking place in the [bitcoin design Slack](http://bitcoindesigners.org) (#bitcoin-design-guide).

> The [Bitcoin Design Guide](https://bitcoin.design/guide/) is a free open-source community resource that helps designers, developers and others working on non-custodial bitcoin-products to create better experiences, faster. We hope that, over time, it will cover all relevant types of products, including consumer wallets, merchant interactions, exchanges and more. Better products and experiences should ultimately make it more appealing for anyone to own and use bitcoin.

Reviewer 4: ```suggestion
The [Bitcoin Design Guide](https://bitcoin.design/guide/) is a free open-source community resource that helps designers, developers and others working on non-custodial bitcoin-products to create better experiences faster. We hope that, over time, it will cover all relevant types of products, including consumer wallets, merchant interactions, exchanges and more. Better products and experiences should ultimately make it more appealing for anyone to own and use bitcoin.
```

> An equally important goal is that the process of creating this guide nurtures an open-source bitcoin design community that can carry this, and other projects forward longer term. The [bitcoin design Slack](http://bitcoindesigners.org) and the [bitcoin.design](http://bitcoin.design) website are the first step in this direction.

Reviewer 3: ```suggestion
An equally important goal is that the process of creating this guide nurtures an open-source Bitcoin design community that can carry this, and other projects forward longer term. The [Bitcoin Design Slack](http://bitcoindesigners.org) and the [bitcoin.design](https://bitcoin.design) website are the first steps in this direction.
```

---

## Process and how to contribute

> The process of creating the guide is meant to be as open and decentralized as possible, resulting in a sustainable model to run projects with the community. Most of the discussion happen in the [bitcoin design Slack](http://bitcoindesigners.org), in the #bitcoin-design-guide, and here on Github in [Issues](https://github.com/BitcoinDesign/Guide/issues), [Pull requests](https://github.com/BitcoinDesign/Guide/pulls) and [Discussions](https://github.com/BitcoinDesign/Guide/discussions).

Reviewer: ```suggestion
The process of creating the guide is meant to be as open and decentralized as possible, resulting in a sustainable model to run projects with the community. Most of the discussion happens in the [bitcoin design Slack](http://bitcoindesigners.org), in the #bitcoin-design-guide, and here on GitHub in [Issues](https://github.com/BitcoinDesign/Guide/issues), [Pull requests](https://github.com/BitcoinDesign/Guide/pulls) and [Discussions](https://github.com/BitcoinDesign/Guide/discussions).
```

> The process of creating the guide is meant to be as open and decentralized as possible, resulting in a sustainable model to run projects with the community. Most of the discussion happens in the [bitcoin design Slack](http://bitcoindesigners.org), in the #bitcoin-design-guide, and here on GitHub in [Issues](https://github.com/BitcoinDesign/Guide/issues), [Pull requests](https://github.com/BitcoinDesign/Guide/pulls) and [Discussions](https://github.com/BitcoinDesign/Guide/discussions).

Reviewer 3: ```suggestion
The process of creating the guide is meant to be as open and decentralized as possible, resulting in a sustainable model to run projects with the community. Most of the discussion happens in the [Bitcoin Design Slack](http://bitcoindesigners.org), in the #bitcoin-design-guide, and here on GitHub in [Issues](https://github.com/BitcoinDesign/Guide/issues), [Pull requests](https://github.com/BitcoinDesign/Guide/pulls) and [Discussions](https://github.com/BitcoinDesign/Guide/discussions).
```

> You can read more about [how to contribute](https://bitcoin.design/guide/contribute/)

Reviewer 3: ```suggestion
You can read more about [how to contribute](https://bitcoin.design/guide/contribute/).
```

## Guide Content

> The guide is meant to be a simple starting-point for anyone looking to build bitcoin-based applications, while providing deeper topics and value for more experienced bitcoin designers. Bitcoin is the platform, and the guide should provide information and resources that should be consistent across applications built on top of it.

Reviewer 3: ```suggestion
The guide is meant to be a simple starting-point for anyone looking to build Bitcoin-based applications, while providing deeper topics and value for more experienced Bitcoin designers. Bitcoin is the platform, and the guide should provide information and resources that should be consistent across applications built on top of it.
```

    *   Onboarding users to bitcoin
        *   Key concepts users should be aware of
        *   Recommended actions on first-run
    *   [Private key management](https://bitcoin.design/guide/private-key-management/introduction/)
        *   Private key schemes
        *   Personal schemes
        *   Shared schemes
        *   Principles
        *   Case studies
    *   Payments and transactions - [Discussion about WIP](https://github.com/BitcoinDesign/Guide/discussions/98)
        *   Transactions overview
        *   Sending bitcoin
        *   Receiving bitcoin
        *   Transaction privacy

>         *   Bitcoin transactiosn in depth

Reviewer 4: ```suggestion
        *   Bitcoin transaction in depth
```

---

## Origin and history

The process to create the guide started in the summer of 2020. The first drafts of this document first existed as Google Docs, here are [Version 1](https://docs.google.com/document/d/1omAxwvCSRlo_u5UL3ThTXFhNccDuN7GJOi4RlZfk--w/edit#heading=h.75nvyav1r98b) and [Version 2](https://docs.google.com/document/d/1YiYeRIybGmxmErCOI4Jc8Qajz3JGM1JYVfUtpzyCzSk/edit?usp=sharing) for reference.

---

> ## How to build and run the site locally

Reviewer: Just a note that altering this will require a link change in #113 specifically [here](https://deploy-preview-113--sad-borg-390916.netlify.app/guide/contribute/illustration-guidelines/#file-size-optimization)

Author: Updated that link now that #113 is merged.

You'll want to run the site locally to test your changes.

**Make sure you have Ruby and Bundler installed:**

1. Check that Ruby is installed (check with `ruby -v`). Must be 2.2.5 or higher. Install instructions can be found at https://www.ruby-lang.org/en/documentation/installation/.
2. If you don't already have Bundler (check with `which bundle`), you can install by following the instructions at https://bundler.io.

**Clone the source code, build, and serve:**

3. Clone this repository and `cd` into it.
4. Run `bundle install`
5. Run `bundle exec jekyll serve`
6. Browse the site at http://127.0.0.1:4000/

> To run on gh-pages, uncomment the base_url variable on _config.yml

Reviewer 3: ```suggestion
To run on gh-pages, uncomment the `base_url` variable in `_config.yml`.
```
